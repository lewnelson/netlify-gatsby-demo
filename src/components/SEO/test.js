import React from 'react'
import SEO from './'
import { shallow } from 'enzyme'
import { useStaticQuery } from 'gatsby'

function getWrapper (propOverrides) {
  return shallow(
    <SEO
      description='description'
      lang='en-GB'
      meta={[]}
      keywords={[ 'unit', 'testing' ]}
      title='testing'
      {...propOverrides}
    />
  )
}

describe('<SEO />', () => {
  beforeEach(() => {
    useStaticQuery.mockReturnValue({
      site: {
        siteMetadata: {
          title: 'Site title',
          description: 'Site description',
          author: 'Lewis Nelson <lewis@lewnelson.com>'
        }
      }
    })
  })

  test('it renders correctly', () => {
    expect(getWrapper()).toMatchSnapshot()
  })
})
