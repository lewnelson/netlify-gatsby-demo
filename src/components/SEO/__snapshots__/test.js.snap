// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SEO /> it renders correctly 1`] = `ShallowWrapper {}`;
